gemfirexd/client/src/main/java/com/pivotal/gemfirexd/internal/client/net/NetConnection.java

```diff
@@ -3764,6 +3765,14 @@ public void setServerVersion(Version serverVersion) {
     this.serverVersion = serverVersion;
   }
 
+  public boolean isSnappyStoreConnection() {
+    return this.snappyStoreConnection;
+  }
+
+  public boolean setSnappyStoreConnection(boolean snappyStoreConnection) {
+    return this.snappyStoreConnection = snappyStoreConnection;
+  }
```

Comment:
I think we should pass in the url piggy backing on augmentedProperties and use that within all overloads of NetConnection instead of handling it from calling point. That way ClientDataSource etc calling points will get reviewed and hard coded partial string like "snappy" matching will go away.